ecs-agent/tmds/utils/netconfig/netconfig_linux.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Q: Should we have the following here?
```
//go:build linux
// +build linux
```

Reply:
I saw we had the following below:
```
//go:build !windows
// +build !windows
```
Is this equivalent to specifying to only build for Linux?

Reply:
I will move the tags to the top. Generally these two are not equivalent because there could be bsd and darwin and both are not windows. I will update it to be linux explicitly.